src/components/landing-page/cd.landing-page.vue

```diff
@@ -186,17 +271,17 @@ class App extends Vue {
 
     const schema: SchemaDefinition = await import(
       /* @vite-ignore */
-      `@/schemas/hydroshare/edit_schema.json`
+      `@/schemas/hydroshare/schema.json`
```

Comment:
here you're using `schema.json` instead of the `edit_schema.json`
So this will cause your earlier work to get edit functionality for the `title` and `abstract` to no longer function. I think you're causing a regression of your own work here...